documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -145,6 +145,27 @@ As of January 2019, the following Postgres versions on RDS come with an up-to-da
 * Postgres 11: any version
 ====
 
+[[postgresql-on-azure]]
+=== PostgreSQL on Azure
+
+It is possible to use Debezium with (https://docs.microsoft.com/azure/postgresql/?WT.mc_id=debezium-docs-abhishgu)[Azure Database for PostgreSQL] that supports `wal2json`, `pgoutput` and `test_decoding` output plugins.
```

Comment:
Debezium does not support the `test_decoding` plug-in. And let's remove the referrer query param (here and  below) :)

Reply:
updated the doc, removed referrer params